vault/metadata.csv

```diff
@@ -543,6 +548,11 @@ vault.vault.swift.delete.sum,gauge,,millisecond,,Duration of a DELETE operation
 vault.vault.swift.list.count,count,,,,Duration of a LIST operation against the Swift storage backend count,0,vault,
 vault.vault.swift.list.quantile,gauge,,millisecond,,Duration of a LIST operation against the Swift storage backend quantile,0,vault,
 vault.vault.swift.list.sum,gauge,,millisecond,,Duration of a LIST operation against the Swift storage backend sum,0,vault,
+vault.vault.token.creation,count,,,,A new service or batch token was created. (Name chosen to be distinct from vault.token.create - an existing sample metric.),0,vault,
```

Comment:
What are the possible values for this metric? It seems like it is a boolean on if a new service or batch token was created, although the type is `count` so I wasn't sure.